partcad/src/partcad/ai_google.py

```
@@ -21,6 +20,7 @@
 # import google.api_core.exceptions
 google_api_core_exceptions = None
 
+from .ai_feature_file import AiContentProcessor
```

Ruff (0.8.2)

Module level import not at top of file

(E402)